plugins/flatpak/gs-flatpak-transaction.c

```diff
@@ -348,11 +373,28 @@ _transaction_add_new_remote (FlatpakTransaction *transaction,
 	return FALSE;
 }
 
+static void
+gs_flatpak_transaction_set_property (GObject *object, guint prop_id, const GValue *value, GParamSpec *pspec)
+{
+	FlatpakTransaction *transaction = FLATPAK_TRANSACTION (object);
+
+	switch ((GsFlatpakTransactionProperty) prop_id) {
```

Comment:
nit: the cast should not be needed here.

Reply:
Please refer to the @pwithnall  advise [here](https://gitlab.gnome.org/GNOME/gnome-software/merge_requests/331#note_619942)

Reply:
The cast is not needed, but gives us additional checking from `-Wswitch-enum` when that’s enabled. I think it should be left in.